lifecycle/support/infrastructure-creation-policy.md

```diff
@@ -0,0 +1,90 @@
+
+3. Deployment and Testing:
+   - Developers use CircleCI and GitHub for managing deployments, following the defined deployment processes and workflows.
+   - Support Team manage and maintain the deployment pipeline making sure it is up to date with current organisations processes.
+   - Perform comprehensive testing of the infrastructure to ensure functionality, compatibility, and adherence to project requirements.
+   - Use loader.io to stress test the infrastructure, identifying potential bottlenecks and performance issues.
+
+4. Security and Compliance:
+   - Use Detectify to scan for OWASP security issues, performing regular security scans on the infrastructure.
+   - Ensure that AWS Shield is set up and active across all EC2 servers to provide protection against DDoS attacks.
+   - Utilize third-party provider Zoonu for pentesting services, when appropriate and requested by clients.
+   - Collaborate with security personnel to address and resolve any security concerns or vulnerabilities.
+
+5. Pre-Launch Checklist:
+   - Before launching the infrastructure, perform a manual pre-launch checklist to ensure all necessary configurations and checks are completed.
+   - Review the infrastructure setup, security measures, backup and recovery procedures, and monitoring capabilities.
+   - Verify compliance with industry regulations and internal policies.
+
+## Infrastructure Modification Process
+
+1. Request:
+   - Developers or stakeholders request investigation to be carried out on the current infrastructure to determine the reason behind any erronous behaviour of a website that cannot be replicated by the developers.
+   - If necessary for the Support Team to carry out a change in infrastructure, both the finance team, and stakeholders must be notified.
+   - If there is a cost increase to the infrastructure then approval must be given by the stakeholders.
+
+2. Provisioning:
+   - Dependent on the magnituted of the infrastructure change, the work may need to be scheduled where no developement work can be deployed to the hosting structure.
+   - New infrastructure must be previsioned in parallel and transferring domain name must impact and involve the client as little as possible.
+
+3. Testing:
+   - Before finalising the change in infrastructure and retiring any old infrastructure, a comprehensive smoke test must be performed to make sure there are no issues.
+   - Deployment configurations need to be modified, where applicable, and tested with the new the infrastructure to ensure development can continue.
+   - If issues are discovered with the new infrastructure post transferring of domain name, the domain name must be transferred back to the old infrastructure as to not impact business.
+
+4. Logging of Change:
+   - After the change has been completed a change log entry must be added to the Infrastructure Change Log Google form to document the change for the fiance team to review.

```

Comment:
I've added in this section about modification to infrastructure. I had to manually write it because Chat GPT wasn't playing ball. You probably want to put it into GPT and tell it to write it so it's less butt.